src/QsCompiler/DataStructures/SyntaxExtensions.fs

```diff
@@ -311,10 +311,9 @@ type QsStatement with
                     (match s.Default with Null -> Seq.empty | Value v -> upcast v.Body.Statements))
             | QsForStatement s -> upcast s.Body.Statements
             | QsWhileStatement s -> upcast s.Body.Statements
-            | QsConjugation s -> Seq.append s.OuterTransformation.Body.Statements s.InnerTransformation.Body.Statements // FIXME: not sure this folder makes sense...
+            | QsConjugation s -> Seq.append s.OuterTransformation.Body.Statements s.InnerTransformation.Body.Statements
```

Comment:
The thing with the folder here that at the very least needs to be documented in the doc comment is that it is not clear to me how it should behave: In the case of the conjugation, the adjoint of the outer block is implicit. If you have something like a statement counter or similar, should it count all three blocks or just two?

Reply:
Added documentation in latest commit